package.json

```diff
@@ -355,7 +355,7 @@
     "css-loader": "^6.7.1",
     "deepdash": "^5.3.9",
     "dompurify": "^2.3.8",
```

Comment:
Can you also bump the `@types/node` package to the one for node 16?

Reply:
Good catch, fixed.